packages/cli-kit/src/log.ts

```diff
@@ -120,11 +138,12 @@ function getLogFilePath(options: {logDir?: string; override?: boolean} = {}) {
 
 // Shaves off older log lines if logs are over maxLogFileSize long.
 async function truncateLogs(logFile: string) {
-  if (fileSizeSync(logFile) < maxLogFileSize) {
+  const fileSize = fileSizeSync(logFile)
```

Comment:
If this is an async function, I believe it would be preferable to await an async filesize function.

In the past, this wasn't enclosed in an async function, so it made sense to do sync.